helper/resource/testing.go

```diff
@@ -1031,6 +1031,36 @@ func testCheckResourceAttrPair(isFirst *terraform.InstanceState, nameFirst strin
 	return nil
 }
 
+// TestRangeLengthResourceAttr is a TestCheckFunc which checks that the length of the value
+// in state for the given name/key is within an expected (closed) range
```

Comment:
I'm not sure if it matters in this repo, but the last linter PR I looked at wanted `.`'s at the end of comments.